paper-swatch-picker.html

```diff
@@ -132,14 +132,9 @@
     Polymer({
       is: 'paper-swatch-picker',
 
-      hostAttributes: {
-        'tabindex': 0
-      },
+      hostAttributes: {'tabindex': 0},
 
-      listeners: {
-        'paper-dropdown-open': '_onOpen',
-        'iron-select': '_onColorTap'
-      },
+      listeners: {'paper-dropdown-open': '_onOpen', 'iron-select': '_onColorTap'},
```

Comment:
Yuck, really?

Reply:
Cannot enforce a force newline unless it ends with a comma or is longer than 80 cols. 

e.g. `listeners: {'paper-dropdown-open': '_onOpen', 'iron-select': '_onColorTap',}`

Reply:
That is google style guide to include them actually: https://google.github.io/styleguide/jsguide.html#features-objects-use-trailing-comma